js/constructor.js

```diff
@@ -45,34 +43,21 @@ Store.prototype.render = function() {
 
     for(var i=0; i<hours.length; i++){
         tdEl = document.createElement('td'); // create td
-        tdEl.textContent = this.cookieSoldDay[i]; // give td content (Name for an individual store)
+        tdEl.textContent = this.cookieSoldDay[i]; // give td content (Cookies Sold for an individual store)
         trEl.appendChild(tdEl); // append the td
     }
     storeTable.appendChild(trEl); // append the tr
 }
-function renderAllStores(){
-    stores.innerHTML = '';
-    for(var i=0; i<patStores.length; i++){
-        stores.appendChild(patStores[i].render());
-    }
-}
-//Inputs for Stores and Data
-new Store('Alki Store', 2, 16, 4.6);
-new Store('Pike Store', 23, 65, 6.3,);
-new Store('SeaTac Store', 3, 24, 1.2);
-new Store('Capital Hill Seattle', 23, 65, 6.3);
-new Store('Seattle Center Store', 11, 38, 3.7);
-new Store('Sean Place', 10, 50, 5.5);
 //Handler for Submission Form
 function handleDataSubmit(event){
     event.preventDefault();
     if (!event.target.storeLocation.value||!event.target.minCust.value||!event.target.maxCust.value||!event.target.avgSale.value){
         return alert('Fields cannot be empty!');
     }
     var location = event.target.storeLocation.value;
-    var min = event.target.minCust.value;
-    var max = event.target.maxCust.value;
-    var avg = event.target.avgSale.value;
+    var min = parseInt(event.target.minCust.value);
+    var max = parseInt(event.target.maxCust.value);
+    var avg = parseInt(event.target.avgSale.value);
```

Comment:
Since the user should be able to enter a decimal value for the average, you should use `parseFloat()` instead of `parseInt()`.